doc/specifications/rest/REST-API-V2.rst

```diff
@@ -264,6 +305,14 @@ Resource                                                          POST
 /content/urlwildcards/<ID>                                        .                   get url wildcard        .                            delete url wc.
 ================================================================= =================== ======================= ============================ ================ ==============
 
+the following alias URIs are supported:
+
+=============================================================== ===============================
+Alias URI                                                       Target
+--------------------------------------------------------------- -------------------------------
+/content/objects/_by_remote_id/<remote-ID>                      /content/objects/<ID>
```

Comment:
Weren't this supposed to use same form as `/content/types/?identifier=article` referred to above?